man/systemd.unit.xml

```diff
@@ -1220,9 +1220,98 @@
         files. This functionality should not be used in normal
         units.</para></listitem>
       </varlistentry>
-
     </variablelist>
+  </refsect1>
+
+  <refsect1>
+    <title>Mapping of unit settings to their inverses</title>
+
+    <para>Unit settings that create a relationship with a second unit usually show up
+    in properties of both units, for example in <command>systemctl show</command>
+    output. This table lists the pairs of properties that are shown on two units
+    which are connected through some dependency.</para>
+
+    <table>
+      <title>
+        "Forward" and "reverse" unit settings
+      </title>
 
+      <tgroup cols='2'>
+        <colspec colname='forward' />
+        <colspec colname='reverse' />
+        <thead>
+          <row>
+            <entry>"Forward" setting</entry>
+            <entry>"Reverse" setting</entry>
```

Comment:
so what is this table supposed to show precisely? If the left column is supposed to show all deps the user can specify, then it isn't complete, it needs to list both Before= → After= and After= → Before after all.